commands/customObject/create.ts

```diff
@@ -15,44 +15,48 @@ const { i18n } = require('../../lib/lang');
 const i18nKey = 'commands.customObject.subcommands.create';
 const { EXIT_CODES } = require('../../lib/enums/exitCodes');
 
-exports.command = 'create <name> <definition>';
+exports.command = 'create [name]';
 exports.describe = i18n(`${i18nKey}.describe`);
 
 exports.handler = async options => {
-  const { definition, name, derivedAccountId } = options;
+  const { path, name: providedName, derivedAccountId } = options;
 
   await loadAndValidateOptions(options);
 
   trackCommandUsage('custom-object-batch-create', null, derivedAccountId);
 
-  const filePath = getAbsoluteFilePath(definition);
+  const filePath = getAbsoluteFilePath(path);
   const objectJson = checkAndConvertToJson(filePath);
 
   if (!objectJson) {
     process.exit(EXIT_CODES.ERROR);
   }
 
+  const name =
+    providedName || (await inputPrompt(i18n(`${i18nKey}.inputSchema`)));
+
   try {
     await batchCreateObjects(derivedAccountId, name, objectJson);
     logger.success(i18n(`${i18nKey}.success.objectsCreated`));
   } catch (e) {
     logError(e, { accountId: derivedAccountId });
     logger.error(
       i18n(`${i18nKey}.errors.creationFailed`, {
-        definition,
+        definition: path,
       })
     );
   }
 };
 
 exports.builder = yargs => {
-  yargs.positional('name', {
-    describe: i18n(`${i18nKey}.positionals.name.describe`),
-    type: 'string',
-  });
-
-  yargs.positional('definition', {
-    describe: i18n(`${i18nKey}.positionals.definition.describe`),
-    type: 'string',
-  });
+  yargs
+    .positional('name', {
+      describe: i18n(`${i18nKey}.positionals.name.describe`),
+      type: 'string',
+    })
+    .option('path', {
```

Comment:
Should there be a prompt fort `path` since it's required? (I saw this wasn't mentioned in the issue so probably not, just double checking)

Reply:
Probably a good call to add one.  I'll update it 👍

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1261/commits/55b705e95243103a419bf87e4dafa3a0f3d64080